### DISCUSSION TITLE
Cucumber dryRun support in webdriver IO

### DISCUSSION THREAD

**Discussion by dibyaranjan-pg** [webdriverio/webdriverio#13786] [Q&A]
Hi 
I have a webdriverIo project for mobile automation with Cucumber. I want to run `dry-run` to generate a report without executing test scripts.  But without capabilities in wdio.config.js, wdio is not able to find specs. Anyone faced similar issue. 

Thanks in advance for any help you can provide.

**Comment by christian-bromann** [comment]
I don't think WebdriverIO has support to do a dry-run. Alternatively you can use the Cucumber CLI to point to the same feature file with the dry run flag.
